src/routes/clientRoutes.js

```diff
@@ -39,4 +44,6 @@ router.get(
 
 router.get('/chefs/:id', isUserCreatedClient, clientController.getChefById);
 
+router.post('/search', isUserCreatedClient, searchController.searchDish);
+
```

Comment:
We are getting dishes when we search for a dish right? so This should be a get request not post request.